SnowRemovalDB/1 tables.sql

```diff
@@ -0,0 +1,23 @@
+use SnowRemovalDB
+go
+
+drop table if exists Client
+go
+
+create table dbo.Client(
+    ClientId int not null identity primary key,
+    FirstName varchar(30) not null constraint ck_Client_FirstName_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(30) not null constraint ck_Client_LastName_cannot_be_blank check(LastName <> ''),
+    Address varchar(50) not null constraint ck_Client_Address_cannot_be_blank check(Address <> ''),
```

Comment:
change column name to be more precise as there are more columns related to client's address.

Reply:
great!